Index: src/main/java/com/example/demo/persistance/entity/SizeCategory.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.demo.persistance.entity;\n\nimport com.example.demo.persistance.entity.enums.SizeCategoryEnum;\nimport jakarta.persistence.*;\nimport jakarta.validation.constraints.NotNull;\nimport lombok.Getter;\nimport lombok.Setter;\n\n@Getter\n@Setter\n@Entity\npublic class SizeCategory {\n    @Column(name = \"size_category\")\n    @Id\n    @GeneratedValue(strategy = GenerationType.IDENTITY)\n    private int sizeCategoryID;\n    @Column(name = \"category_name\")\n    @Enumerated(EnumType.STRING)\n    private SizeCategoryEnum category;\n    @Column(name = \"category_description\")\n    private String categoryDescription;\n\n    public SizeCategory() {\n    }\n\n\n}
===================================================================
diff --git a/src/main/java/com/example/demo/persistance/entity/SizeCategory.java b/src/main/java/com/example/demo/persistance/entity/SizeCategory.java
--- a/src/main/java/com/example/demo/persistance/entity/SizeCategory.java	
+++ b/src/main/java/com/example/demo/persistance/entity/SizeCategory.java	
@@ -6,8 +6,9 @@
 import lombok.Getter;
 import lombok.Setter;
 
-@Getter
+
 @Setter
+@Getter
 @Entity
 public class SizeCategory {
     @Column(name = "size_category")
Index: src/main/java/com/example/demo/persistance/entity/Shelters.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.demo.persistance.entity;\n\nimport jakarta.persistence.*;\nimport lombok.Getter;\nimport lombok.Setter;\n\nimport javax.validation.constraints.NotNull;\n\n@Getter\n@Setter\n@Entity\n@Table(name = \"shelters\")\npublic class Shelters {\n\n    @Column(name = \"shelter_id\")\n    @Id\n    @GeneratedValue(strategy = GenerationType.IDENTITY)\n    private int shelterID;\n\n    @Column(name = \"shelter_name\")\n    @NotNull\n    private String shelterName;\n\n    @Column(name = \"shelter_city\")\n    @NotNull\n    private String shelterCity;\n\n    @Column(name = \"shelter_address\")\n    @NotNull\n    private String shelterAddress;\n\n    @Column(name = \"shelter_phone\")\n    private String shelterPhone;\n\n    @Column(name = \"shelter_email\")\n    private String shelterEmail;\n\n    public Shelters() {\n    }\n\n}\n
===================================================================
diff --git a/src/main/java/com/example/demo/persistance/entity/Shelters.java b/src/main/java/com/example/demo/persistance/entity/Shelters.java
--- a/src/main/java/com/example/demo/persistance/entity/Shelters.java	
+++ b/src/main/java/com/example/demo/persistance/entity/Shelters.java	
@@ -1,6 +1,9 @@
 package com.example.demo.persistance.entity;
 
-import jakarta.persistence.*;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
 import lombok.Getter;
 import lombok.Setter;
 
@@ -14,7 +17,6 @@
 
     @Column(name = "shelter_id")
     @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private int shelterID;
 
     @Column(name = "shelter_name")
